Auth0/SafariWebAuth.swift

```diff
@@ -23,6 +23,12 @@
 import UIKit
 import SafariServices
 
+#if swift(>=4.2)
+public typealias OpenURLOptionsKey = UIApplication.OpenURLOptionsKey
```

Comment:
I don't think setting a public `typealias` is a good idea as it can conflict with others. Perhaps it should be inside a `struct` or something, e.g.
```Swift
struct Auth0Application {
   #if swift
   ...
   #endif
}
```
Would that be possible? That way we can keep backwards compatibility without risking others conflicting, in case they try to use the same trick.

 It also doesn't make sense to define it in this class instead of in `Auth0.swift`, for example.

Thoughts?

Reply:
It has to be public due to some of the methods that include it as a param type being public. I could prefix it to avoid an unlikely collision e.g. `Auth0URLOptionsKey`

Auth0.swift doesn't include UIKit as it's used cross platform, UIKit is for iOS and the only areas that use this are in the files that are only included for iOS.